src/lib/cache/subCache.cpp

```diff
@@ -99,6 +100,26 @@ entityId(_entityId), entityType(_entityType)
   {
     entityIdPatternToBeFreed = false;
   }
+
+  if (isTypePattern)
+  {
+    // FIXME P5: recomp error should be captured? have a look to other usages of regcomp()
```

Comment:
Is this FIXME correct? As far as I understand, the if and `alarmMgr.badInput(clientIp, "invalid regular expression for typePattern")` is a way of implementing such regcomp error capture.